### PR TITLE
Class checking: "instanceof"

### PR DESCRIPTION
**Pull request by mahdiHash**
Please review and comment/commit if anything is wrong.

### CODE REVIEW

File: 1-js/09-classes/06-instanceof/article.md [hunk 1]
```diff
@@ -93,76 +93,76 @@ The algorithm of `obj instanceof Class` works roughly as follows:
...
 
-// ...not a rabbit any more!
+// !نیست (rabbit) دیگر یک خرگوش
 *!*
 alert( rabbit instanceof Rabbit ); // false
 */!*
 ```
 
-## Bonus: Object.prototype.toString for the type
+## کمک: متد Object.prototype.toString برای نوع
```
Comment (mahdiHash) [inline 1-js/09-classes/06-instanceof/article.md:127]: This line needs more attention (translation of "bonus")

File: 1-js/09-classes/06-instanceof/article.md [hunk 2]
```diff
@@ -186,33 +186,33 @@ let user = {
...
 
 alert( {}.toString.call(window) ); // [object Window]
 alert( {}.toString.call(new XMLHttpRequest()) ); // [object XMLHttpRequest]
 ```
 
-As you can see, the result is exactly `Symbol.toStringTag` (if exists), wrapped into `[object ...]`.
+همانطور که می‌بینید، نتیجه دقیقا `Symbol.toStringTag` (اگر وجود داشته باشد) جایگذاری شده درون `[object ...]` است.
 
-At the end we have "typeof on steroids" that not only works for primitive data types, but also for built-in objects and even can be customized.
+در نهایت ما «انواعی از استروئیدها» را داریم که نه تنها برای انواع داده اصلی کار می‌کند، بلکه برای شیءهای درون‌ساخت هم کار می‌کند و حتی می‌تواند شخصی‌سازی شود.
```
Comment (mahdiHash) [inline 1-js/09-classes/06-instanceof/article.md:202]: This line needs more attention (translation of "types on steroids"
I guess "on" is a typo. Maybe "of" makes more sense?

File: 1-js/09-classes/06-instanceof/article.md [hunk 3]
```diff
@@ -186,33 +186,33 @@ let user = {
...
-| `instanceof`  | objects     |  true/false   |
+| `typeof`      | مقدارهای اصلی  |  رشته       |
+| `{}.toString` | مقدارهای اصلی، شیءهای درون‌ساخت، شیءها شامل `Symbol.toStringTag`   |       رشته |
+| `instanceof`  | شیءها     |  true/false   |
 
-As we can see, `{}.toString` is technically a "more advanced" `typeof`.
+همانطور که می‌بینید، `{}.toString` از لحاظ فنی یک `typeof` «پیشرفته‌تر» است.
 
-And `instanceof` operator really shines when we are working with a class hierarchy and want to check for the class taking into account inheritance.
+و زمانی که با سلسله‌ای از کلاس‌ها کار می‌کنیم و می‌خواهیم بررسی کنیم که کلاس در ارث‌بری وجود دارد یا نه، عملگر `instanceof` واقعا می‌درخشد.
```
Comment (mahdiHash) [inline 1-js/09-classes/06-instanceof/article.md:218]: This line needs more attention.

File: 1-js/09-classes/06-instanceof/article.md (hunk 1 above)
Comment (mahdyar) [reply 1-js/09-classes/06-instanceof/article.md:127]:
```suggestion
## راهنمایی: متد Object.prototype.toString برای نوع
```

**Comment by mahdiHash** [comment]
Thank you @mahdyar 
/done

File: 1-js/09-classes/06-instanceof/article.md (hunk 2 above)
Comment (mahdyar) [reply 1-js/09-classes/06-instanceof/article.md:202]: LGTM!

File: 1-js/09-classes/06-instanceof/article.md (hunk 3 above)
Comment (mahdyar) [reply 1-js/09-classes/06-instanceof/article.md:218]:
```suggestion
 زمانی که با سلسله‌ای از کلاس‌ها کار می‌کنیم و می‌خواهیم بررسی کنیم که کلاس در ارث‌بری وجود دارد یا نه، عملگر `instanceof` واقعا می‌درخشد.
```

**Review by mahdyar — CHANGES_REQUESTED** [review]
Please review the suggested changes and request for another review.

**Comment by mahdiHash** [comment]
Thank you @mahdyar 
/done

**Comment by mahdyar** [comment]
Thank you, @mahdiHash!

**Review by mahdyar — APPROVED** [review]

**Pull request merged**